src/Database/Schema/Blueprint.php

```diff
@@ -241,6 +241,16 @@ public function integer($name, $autoIncrement = false, $unsigned = false): Prope
         return $this->addColumn('integer', $name);
     }
 
+    /**
+     * @param string $name
+     *
+     * @return PropertyDefinition
+     */
+    public function long($name): PropertyDefinition
```

Comment:
You need to move this function because the methods are alphabetised.

And type-hint!
```suggestion
    public function long(string $name): PropertyDefinition
```